libklog/libklog.h

```diff
@@ -25,10 +25,13 @@ typedef void (*libklog_write_t)(const char *buf, size_t size);
 int libklog_ctrlHandle(uint32_t port, msg_t *msg, msg_rid_t rid);
 
 
+void libklog_enable(int enable);
+
+
 int libklog_ctrlRegister(oid_t *oid);
 
 
-int libklog_init(libklog_write_t clbk);
+int libklog_init(libklog_write_t clbk, int createDevs);
```

Comment:
Can you do this without modifying the existing API? Perhaps introducing `libklog_attach(libklog_write_t clbk)` would be a better approach?

Reply:
Ok, but I didn;t like `attach` so I went ahead with `initNoDev`